analyses/tmb-compare-tcga/TMB_d3b_code/code/01_calculate_tmb_targetflexible_withbothexperstrt_and_cohort.py

```diff
@@ -0,0 +1,235 @@
+##  This script takes MAF file and  metadata  file with experimental strategy
+#   and target config file and computes TMB  scores for every samplename
+####### Author: Teja Koganti ##################
+# Inputs  - 1. MAF file - Consensus should be performed prior to using as INputs
+#               Must follow the MAF format. It's okay  if the first
+#               commented line is absent
+########### 2. metadata file - This should have  disease column and
+#               experimental strategy. Sample should not be  repeated under
+#               different experimental strategies
+########### 3. target config file - This should have two columns,
+#               one with experimental strategy in metadata  and another with the
+#               corresponding BED file for processing variant counts
+
+
+# python3 01_calculate_tmb_targetflexible.py \
```

Comment:
note that this does not match the file name (Personally, I like the shorter version here better.)